src/test/FuzzerImpl.cpp

```diff
@@ -81,6 +82,30 @@ getShortKey(AssetCode12 const& code)
     return code.data()[0] % NUMBER_OF_ASSETS_TO_USE;
 }
 
+uint8_t constexpr ENCODE_ASSET_ISSUER_SHIFT = 4;
+uint8_t constexpr ENCODE_ASSET_ISSUER_MASK =
+    (1 << (8 - ENCODE_ASSET_ISSUER_SHIFT)) - 1;
+uint8_t constexpr ENCODE_ASSET_CODE_MASK = (1 << ENCODE_ASSET_ISSUER_SHIFT) - 1;
+
+uint8_t
+encodeAsset(uint8_t issuer, uint8_t digit)
+{
```

Comment:
I did the masking with the intent that it would work for any values of either, just that it would produce duplicate encodings for some different pairs.  So there are some assets that the fuzz-generated data would never be decoded to by the unpacker, and there are different assets that would get the same encoding (so that if setup encoded an asset into some byte, and the fuzzer generated a test case where the compact XDR had that byte in an `Asset`, it could decode to something other than the asset that setup had encoded).  But I think that's tolerable and inevitable.  Does that make sense?

Reply:
Yeah that makes sense.